File: Extras/006 - Find Digits/FindDigits.py
def findDigits(n):
  digits = list(filter(lambda d: d > 0, list(map(lambda d: int(d), list(f'{n}')))))
  divisors = list(filter(lambda digit: n % digit == 0, digits))
  
  return len(divisors)


print(findDigits(1012))

File: Extras/006 - Find Digits/FindDigits.js
function findDigits(n) {
  const digits = `${n}`.split('').map(d => parseInt(d)).filter(d => d > 0);
  const divisors = digits.filter(digit => n % digit === 0);

  return divisors.length;
}

console.log(findDigits(1012));
